.editorconfig

```diff
@@ -4,6 +4,7 @@ root = true
 indent_style = space
 indent_size = 4
 continuation_indent_size=4
+ktlint_disabled_rules=trailing-comma-on-call-site,trailing-comma-on-declaration-site
```

Comment:
Why did you disable these rules?

Reply:
to avoid force developers adding commas on call and declaration site: we never force each other to adding trailing commas in pull requests

Reply:
Why not force them to add trailing commas? I think this is a rather useful rule.

Reply:
it's usefulf to add commas when every new argument is on a new line, otherwise it's useless. ktlint doesn't have separate rules for `single line params` vs `multiple lines params`. When (and if) these rules be separated we should be good to force adding the very last comma for multi lines params

Reply:
ktlint does not report a violation if all arguments are on the same line.

Reply:
I still don't think that force people to set commas is good idea. We have public API that is not changing (at least till v3.0) and there's no sense to add end commas on declarative nor call site.  For now no rule setup that divide public API and internal with different settings

Reply:
That is irrelevant, you don't break API by adding trailing commas.

Reply:
sure. But you don't need to add trailing commas: a public API won't be changed so no need to force to put triangling commas